android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotification.java

```diff
@@ -68,6 +68,11 @@ public void onNewIntent(Intent intent) {
             bundle.putBoolean("foreground", false);
             intent.putExtra("notification", bundle);
             notifyNotification(bundle);
+        }else if ( intent.getExtras() != null ) {
+            Bundle bundle = intent.getExtras();
+            bundle.putBoolean("foreground", false);
+            intent.putExtra("notification", bundle);
+            notifyNotification(bundle);
         }
     }
```

Comment:
this fixed a problem when you open the app from a click on a notification the notification wasn't available in js